observability/distr_tracing/distr-tracing-tempo-removing.adoc

```diff
@@ -1,11 +1,16 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="dist-tracing-tempo-removing"]
-= Removing
+= Removing the {DTShortName}
 include::_attributes/common-attributes.adoc[]
 :context: dist-tracing-tempo-removing
 
 toc::[]
 
+[TIP]
+====
+For removing the deprecated and soon to be removed {JaegerShortName}, see xref:../../observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-removing.adoc#dist-tracing-jaeger-removing[Removing] in the {JaegerShortName} documentation.
```

Comment:
> the deprecated and soon to be removed

Reply:
Yes, because Tempo is the replacement product for Jaeger, so readers can confuse the docs and may be looking at the wrong product.